Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v2/FeedOpenCTI_v2.py

```diff
@@ -0,0 +1,488 @@
+
+    if result:
+        readable_output = f'Added {key} successfully.'
+    else:
+        raise DemistoException(f"Can't add {key}.")
+
+    return CommandResults(readable_output=readable_output)
+
+
+def indicator_field_remove_command(client, args: Dict[str, str]) -> CommandResults:
+    """ Remove indicator marking or label from opencti
+
+        Args:
+            client: OpenCTI Client object
+            args: demisto.args()
+
+        Returns:
+            readable_output
+        """
+    indicator_id = args.get("id")
+    # works only with marking and label
+    key = args.get("field")
+    value = args.get("value")
+    result = {}
+
+    if key == 'marking':
+        mark_obj = MarkingDefinition(client)
+        marking = mark_obj.create(definition=value, definition_type='TLP').get('id')
+        result = client.stix_cyber_observable.remove_marking_definition(id=indicator_id, marking_definition_id=marking)
+
+    elif key == 'label':
+        label_obj = Label(client)
+        label_id = label_obj.create(value=value).get('id')
+        result = client.stix_cyber_observable.remove_label(id=indicator_id, label_id=label_id)
+
+    if result:
+        readable_output = 'Field removed successfully.'
+    else:
+        raise DemistoException(f"Can't remove {key}.")
+
+    return CommandResults(readable_output=readable_output)
+
+
+def organization_list_command(client, args) -> CommandResults:
+    """ Get organizations list from opencti
+
+        Args:
+            client: OpenCTI Client object
+            args: demisto.args()
+
+        Returns:
+            readable_output, raw_response
+        """
+    limit = int(args.get('limit', '50'))
+    organizations_list = client.identity.list(types='Organization', first=limit)
+    if organizations_list:
+        organizations = [
+            {'name': organization['name'], 'id': organization['id']}
+            for organization in organizations_list]
+        readable_output = tableToMarkdown('Organizations from OpenCTI', organizations, headerTransform=pascalToSpace)
+        return CommandResults(
+            outputs_prefix='OpenCTI.Organizations',
+            outputs_key_field='id',
+            outputs=organizations,
+            readable_output=readable_output,
+            raw_response=organizations_list
+        )
+    else:
+        return CommandResults(readable_output='No organizations')
+
+
+def organization_create_command(client, args: Dict[str, str]) -> CommandResults:
+    """ Create organization at opencti
+
+        Args:
+            client: OpenCTI Client object
+            args: demisto.args()
+
+        Returns:
+            readable_output, raw_response
+        """
+    name = args.get("name")
+    description = args.get("description")
+    reliability = args.get('reliability')
+    identity = Identity(client)
+    result = identity.create(name=name, type='Organization', x_opencti_reliability=reliability,
+                             description=description)
```

Comment:
Create a function for each that will also contain try-except.

Reply:
which function? I just added try except.